pkg/config/config_template.yaml

```diff
@@ -1143,12 +1143,33 @@ api_key:
   ## Defines rules by which to filter traces based on tags.
   ##  * require - list of key or key/value strings - traces must have those tags in order to be sent to Datadog
   ##  * reject - list of key or key/value strings - traces with these tags are dropped by the Agent
-  ## Note: Rules take into account the intersection of tags defined.
+  ## Please note that:
+  ##   1. Rules take into account the intersection of tags defined.
+  ##   2. When `filter_tags` and `filter_tags_regex` are used at the same time, all rules are united for filtering.
+  ##      In cases where rules in `filter_tags` and `filter_tags_regex` have the same key, the rule from `filter_tags`
+  ##      takes precendence over the rule from `filter_tags_regex`.
+  ##
+  ##      In the case of the following configuration:
+  ##        filter_tags:
+  ##          require: ["foo:bar"]
+  ##        filter_tags_regex:
+  ##          require: ["foo:^bar[0-9]{1}$"]
+  ##      The resulting combined rule will match `foo:bar` and not `foo:bar1`
   #
   # filter_tags:
   #     require: [<LIST_OF_KEY_VALUE_TAGS>]
   #     reject: [<LIST_OF_KEY_VALUE_TAGS>]
 
+  ## @param filter_tags_regex - object - optional
+  ## Defines rules by which to filter traces based on tags with regex pattern for tag values.
+  ##  * require - list of key or key/value regex pattern strings - traces must have those tags in order to be sent to Datadog
+  ##  * reject - list of key or key/value regex pattern strings - traces with these tags are dropped by the Agent
+  ## Note: Rules take into account the intersection of tags defined.
```

Comment:
Since we're not copying al of the docs from `filtering_tags` into the `filtering_tags_regexp` docs, let's just call out that someone should look there for full details about usage.

Reply:
Added a reference to the description of the filtering_tags parameter